satpy/readers/seviri_l2_bufr.py

```diff
@@ -38,8 +37,9 @@
 
 logger = logging.getLogger('BufrProductClasses')
 
+data_centre_dict = {55: {'ssp': 'E0415', 'name': '08'}, 56:  {'ssp': 'E0000', 'name': '09'},
```

Comment:
The official language of satpy is American English, so it should be 'center' (sorry...)

Reply:
fixed